src/amo/middleware/responseTime.js

```diff
@@ -26,14 +26,14 @@ export const responseTime = ({
       // TODO: generate a key based on the rendered component, which I think is
       // in server/base.js -> match() -> renderProps.components
 
-      client.increment(`response_code.${res.statusCode}.count`);
+      client.increment(`response_status.${res.statusCode}.count`);
+      client.increment(`response_method.${req.method}.count`);
 
-      const responseTypeKey = `response.${req.method}`;
-      client.increment(`${responseTypeKey}.count`);
       // The time variable is response time in milleseconds.
-      client.timing(`${responseTypeKey}.time`, time);
+      const responseTimeKey = `response.${req.method}_${res.statusCode}.time`;
+      client.timing(responseTimeKey, time);
```

Comment:
I looked for a similar key in grafana for timing but couldn't find anything... so it's either something entirely different or we don't collect it for addons-server?

Reply:
There is no timing metric for status code AND method on addons-server, just for method (and maybe status code, separately). In addition, I don't think we use the `.time` suffix on addons-server either.